bookkeeper-server/src/main/java/org/apache/bookkeeper/replication/ReplicationWorker.java

```diff
@@ -227,9 +227,12 @@ public void run() {
         workerRunning = true;
         while (workerRunning) {
             try {
-                rereplicate();
+                if (!rereplicate()) {
+                    LOG.warn("failed while replicating fragments");
+                    waitBackOffTime(rwRereplicateBackoffMs);
+                }
             } catch (InterruptedException e) {
-                LOG.info("InterruptedException "
+                LOG.error("InterruptedException "
```

Comment:
Why are you changing this to ERROR?

Reply:
@eolivelli I think this log should be set to error level, it's a Interrupted Exception

Reply:
> Why are you changing this to ERROR?

@eolivelli it may be manual or unexcept interrupted, this log's level should be warn or error, I think.

Reply:
> Why are you changing this to ERROR?

@eolivelli it may be manual or unexcept interrupted, this log's level should be warn or error, I think.

Reply:
> Why are you changing this to ERROR?

@eolivelli it may be manual or unexcept interrupted, this log's level should be warn or error, I think

Reply:
> Why are you changing this to ERROR?

@eolivelli it may be manual or unexcept interrupted, this log's level should be warn or error, I think

Reply:
> Why are you changing this to ERROR?

@eolivelli it may be manual or unexcept interrupted, this log's level should be warn or error, I think

Reply:
> Why are you changing this to ERROR?

@eolivelli it may be manual or unexcept interrupted, this log's level should be warn or error, I think

Reply:
@eolivelli 
it may be manual or unexcept interrupted, this log's level should be warn or error, I think